src/gdpr-utils.js

```diff
@@ -83,9 +83,14 @@ export function hasOptedIn(token, options) {
  */
 export function hasOptedOut(token, options) {
     if (_hasDoNotTrackFlagOn(options)) {
+        console.warn('[Mixpanel SDK] This browser has "Do Not Track" enabled. This will prevent the Mixpanel SDK from sending any data. To ignore the "Do Not Track" browser setting, initialize the Mixpanel instance with the config "ignore_dnt: true"');
```

Comment:
you'll still need to `import { console } from './utils';` to get the debug behavior